students/tony7kh/hw-5/answers.js

```diff
@@ -0,0 +1,36 @@
+//Task 1
+const getMaxNumber = (...number) => result = Math.max(...number);
+console.log(getMaxNumber(2, 0));
+console.log(getMaxNumber(2, 0, 5));
+console.log(getMaxNumber(2, 0, 5, -1, 3, 10, 4));
+
+
+//Task 2
+const joinObj = (obj1, obj2, ...obj) => {
+    return student = Object.assign({}, obj1, obj2, ...obj)
+};
+console.log(joinObj({name: 'Nike'}, {title: 'boss'}, {age: 30},{position: 'CEO'}, {isDefault: false}));
+
+//Task 3
+
+const student = {
+    name: 'Anna',
+    age: 20
+};
+
+Object.defineProperties(student, 
+    {'toString': {
+    enumerable: false,
```

Comment:
```suggestion
```

Reply:
When you use `Object.defineProperties` each props will be `enumerable: false` by default